webpage/style.css

```
@@ -530,14 +535,18 @@ textarea {
 	cursor: pointer;
 }
 
-#servername {
+#channelname, #servername {
 	margin-top: .1in;
 	margin-bottom: .1in;
 	padding-left: 5px;
 	text-overflow: ellipsis;
 	overflow-x: hidden;
 	white-space: nowrap;
-	max-width: 220px;
+}
+
+#serverName {
```

not going to work unless you change it to the following, as there's no element with this ID:
```suggestion
#servername {
```